pabi_procurement/models/purchase_request.py

```diff
@@ -55,6 +55,7 @@ class PurchaseRequest(models.Model):
     currency_id = fields.Many2one(
         'res.currency',
         'Currency',
```

Comment:
For good practice, please use "string" for every fields in all files. i.e.,
string='Currency'

This will help us spot when we forget the set the field label.